org.learningequality.Kolibri.appdata.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.learningequality.Kolibri</id>
```

Comment:
Also please set <launchable> tag pointing to the desktop file name. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#qsr-app-launchable-info